custom_event_checks/opslevel/service_infrastructure_check/aws-postgres-extended-support-check.yml

```diff
@@ -0,0 +1,34 @@
+    | (.rawData.engine_version | tonumber) >= 12 ] | all'
+  message: "{% if check.passed %}\n  ### Check passed\n  \U0001F389 Congratulations!
+    All postgres databases linked to this service are running version 12 or newer.\n{%
+    else %}\n  ### Check failed\n  Service **{{ ctx.alias }}** has postgres databases
+    running on version 11 or older.  These databases need to be upgraded.\n\n  {%-
+    assign resources = data | where: \"name\", ctx.alias  %}\n  {% for resource in
+    resources %}\n  {% assign dbs = resource.relatedResources.nodes | where: \"type\",
+    \"Database\" %}\n  {% for db in dbs %}\n  {% assign pg_db = db.rawData | where:
+    \"engine\", \"aurora-postgresql\" | first %}\n  {% if pg_db %}\n  {% assign engine_version
+    = pg_db.engine_version | plus:0 %}\n  {% if engine_version < 12 %}\n  * {{ pg_db.db_instance_identifier
+    }} - {{ pg_db.engine_version }}\n  {% endif %}\n  {% endif %}\n  {% endfor %}\n
+    \ {% endfor %}\n{% endif %}\n"
+  samplePayload: '[{"id":"Z2lkOi8vb3BzbGV2ZWwvU2VydmljZS85ODQwNA","name":"shopping-cart","aliases":["shopping-cart","shopping-cart-dev","shopping-cart-production","shopping-cart-qa"],"relatedResources":{"nodes":[{"id":"Z2lkOi8vb3BzbGV2ZWwvRW50aXR5T2JqZWN0LzEwNTc3MDE","href":"/catalog/infrastructure/1057701","name":"shopping-cart-dev-1","type":"Database","rawData":{"arn":"arn:aws:rds:us-east-1:987654321234:db:shopping-cart-dev-1","type":"db_instance","engine":"aurora-postgresql","db_name":"shopping_cart","endpoint":{"port":5432,"address":"shopping-cart-dev-1.ck8i1mnqeulo.us-east-1.rds.amazonaws.com","hosted_zone_id":"Z2R2ITUGPM61AM"},"multi_az":false,"tag_list":[{"key":"Project","value":"Partnerships"},{"key":"ManagedBy","value":"terraform"},{"key":"CostCenter","value":"engineering"},{"key":"Accounting","value":"200"},{"key":"Environment","value":"dev"},{"key":"Application","value":"ShoppingCart"},{"key":"AppName","value":"shopping-cart"},{"key":"Name","value":"shopping-cart-dev-1"}],"kms_key_id":"arn:aws:kms:us-east-1:987654321234:key/bda7cdc0-8f2f-435a-a5cc-4bb6e8cbe508","network_type":"IPV4","storage_type":"aurora","backup_target":"region","license_model":"postgresql-license","engine_version":"11.21","promotion_tier":1,"db_instance_arn":"arn:aws:rds:us-east-1:987654321234:db:shopping-cart-dev-1","db_subnet_group":{"vpc_id":"vpc-0b28432c3a2a8bcc2","subnets":[{"subnet_status":"Active","subnet_outpost":{},"subnet_identifier":"subnet-0e01fd24af2db80b7","subnet_availability_zone":{"name":"us-east-1a"}},{"subnet_status":"Active","subnet_outpost":{},"subnet_identifier":"subnet-0d56d3c9c73f08ac4","subnet_availability_zone":{"name":"us-east-1b"}},{"subnet_status":"Active","subnet_outpost":{},"subnet_identifier":"subnet-04e8659eceb5a4362","subnet_availability_zone":{"name":"us-east-1c"}}],"subnet_group_status":"Complete","db_subnet_group_name":"shopping-cart-dev-aurora-postgresql","db_subnet_group_description":"Subnet
```

Comment:
```suggestion
  samplePayload: >-
    '[{"id":"Z2lkOi8vb3BzbGV2ZWwvU2VydmljZS85ODQwNA","name":"shopping-cart","aliases":["shopping-cart","shopping-cart-dev","shopping-cart-production","shopping-cart-qa"],"relatedResources":{"nodes":[{"id":"Z2lkOi8vb3BzbGV2ZWwvRW50aXR5T2JqZWN0LzEwNTc3MDE","href":"/catalog/infrastructure/1057701","name":"shopping-cart-dev-1","type":"Database","rawData":{"arn":"arn:aws:rds:us-east-1:987654321234:db:shopping-cart-dev-1","type":"db_instance","engine":"aurora-postgresql","db_name":"shopping_cart","endpoint":{"port":5432,"address":"shopping-cart-dev-1.ck8i1mnqeulo.us-east-1.rds.amazonaws.com","hosted_zone_id":"Z2R2ITUGPM61AM"},"multi_az":false,"tag_list":[{"key":"Project","value":"Partnerships"},{"key":"ManagedBy","value":"terraform"},{"key":"CostCenter","value":"engineering"},{"key":"Accounting","value":"200"},{"key":"Environment","value":"dev"},{"key":"Application","value":"ShoppingCart"},{"key":"AppName","value":"shopping-cart"},{"key":"Name","value":"shopping-cart-dev-1"}],"kms_key_id":"arn:aws:kms:us-east-1:987654321234:key/bda7cdc0-8f2f-435a-a5cc-4bb6e8cbe508","network_type":"IPV4","storage_type":"aurora","backup_target":"region","license_model":"postgresql-license","engine_version":"11.21","promotion_tier":1,"db_instance_arn":"arn:aws:rds:us-east-1:987654321234:db:shopping-cart-dev-1","db_subnet_group":{"vpc_id":"vpc-0b28432c3a2a8bcc2","subnets":[{"subnet_status":"Active","subnet_outpost":{},"subnet_identifier":"subnet-0e01fd24af2db80b7","subnet_availability_zone":{"name":"us-east-1a"}},{"subnet_status":"Active","subnet_outpost":{},"subnet_identifier":"subnet-0d56d3c9c73f08ac4","subnet_availability_zone":{"name":"us-east-1b"}},{"subnet_status":"Active","subnet_outpost":{},"subnet_identifier":"subnet-04e8659eceb5a4362","subnet_availability_zone":{"name":"us-east-1c"}}],"subnet_group_status":"Complete","db_subnet_group_name":"shopping-cart-dev-aurora-postgresql","db_subnet_group_description":"Subnet
```
Same idea as above. Combines lines under `samplePayload:` into one line because rogue newlines in json can cause issues